python/ccxt/base/precise.py

```diff
@@ -32,6 +32,51 @@ def __init__(self, number, decimals=None):
             self.decimals = decimals
         self.base = 10
 
+    def __add__(self, other):
+        return self.add(other)
+
+    def __sub__(self, other):
+        return self.sub(other)
+
+    def __mul__(self, other):
+        return self.mul(other)
+
+    def __truediv__(self, other):
+        return self.div(other)
+
+    def __mod__(self, other):
+        return self.mod(other)
+
+    def __neg__(self):
+        return self.neg()
+
+    def __abs__(self):
+        return self.abs()
+
+    def __min__(self, other):
+        return self.min(other)
+
+    def __max__(self, other):
+        return self.max(other)
+
+    def __lt__(self, other):
+        return self.lt(other)
+
+    def __le__(self, other):
+        return self.le(other)
+
+    def __gt__(self, other):
+        return self.gt(other)
+
+    def __ge__(self, other):
+        return self.ge(other)
+
+    def __eq__(self, other):
+        if isinstance(other, str):
+            # Allow comparisons with Precise("5") == "5"
+            return str(self) == other
```

Comment:
One can argue about this - it allows for direct string comparison (`Precise("55.5") == "55.5").

I don't think it'll be strictly necessary - but it did simplify some of the tests...